hw/top_earlgrey/chip_earlgrey_cw310_hyperdebug.core

```diff
@@ -0,0 +1,112 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:systems:chip_earlgrey_cw310_hyperdebug:0.1"
+description: "Earl Grey toplevel for the ChipWhisperer CW310 board's hyperdebug variant"
+filesets:
+  files_rtl_cw310:
+    depend:
+      - lowrisc:systems:top_earlgrey:0.1
+      - lowrisc:systems:top_earlgrey_pkg
+      - lowrisc:systems:ast
+      - lowrisc:systems:padring
+    files:
+      - rtl/clkgen_xil7series.sv
+      - rtl/usr_access_xil7series.sv
+      - rtl/autogen/chip_earlgrey_cw310.sv
+    file_type: systemVerilogSource
+
+  files_constraints:
+    files:
+      - data/clocks.xdc
```

Comment:
boy...is this really the only difference compared to the other one?
i wish there was a way to inherit ...

Reply:
We might be able to create a common core, then have the other targets depend on it :thinking:

Reply:
Another direction would be to have two filesets and choose between them within one core file. However, that would mean building the same fusesoc_core twice, each with different parameters. All the output files might then be named similarly...

Reply:
ah i'm not suggesting we really change..i think i'm just lamenting the fact that there isn't a clean way in our existing flows to do this (other than refactoring common cores).

I think this is fine though.  Maybe we could leave a comment on how this is the primary difference with the other? Maybe that's enough.

Reply:
Yeah this is annoying but probably ok for now.
I had to do something similar for the closed source sims where it is not possible to inherit some of the generator and target stuff.